x-pack/plugins/security_solution/server/usage/detections/detection_rule_helpers.ts

```diff
@@ -202,7 +229,7 @@ export const getDetectionRuleMetrics = async (
       body: {
         aggs: {
           detectionAlerts: {
-            terms: { field: 'signal.rule.id.keyword' },
+            terms: { field: 'signal.rule.id' },
```

Comment:
NOTE: I had to change this that was not in the original PR as this looked to not be working in the e2e tests without it.